lnrpc/rpc.proto

```diff
@@ -3461,6 +3472,18 @@ message BakeMacaroonResponse {
     string macaroon = 1;
 }
 
+message ListSweepsRequest {
+}
+
+message ListSweepsResponse {
+    /*
+    The hex encoded transaction ids of the sweeps that our node has broadcast.
```

Comment:
nit: Mention that it's the reversed, hex-encoded string representation (see `OutPoint`).